cli/src/org/partiql/cli/prettyprintAST/PrettyPrinter.kt

```diff
@@ -0,0 +1,508 @@
+package org.partiql.cli.prettyprintAST
+
+import com.amazon.ion.system.IonSystemBuilder
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.syntax.SqlParser
+import org.partiql.pig.runtime.SymbolPrimitive
+
+/**
+ * This class is used to pretty print PIG AST.
+ */
+class PrettyPrinter {
```

Comment:
Does it make sense to move this class and the `RecursiveTree` structure to the `lang` package? I can see some customers are asking for a `PrettyPrinter` that they can consume from within `lang`.

Reply:
My thinking is in that case, users _must_ use CLI for this, even if they don't need to.

Reply:
Seems our customer also asks for it here https://github.com/partiql/partiql-lang-kotlin/issues/552